README.md

```diff
@@ -1,16 +1,81 @@
 NextEuropa User Management
 
+# Change log
+## V 1.2.0
+- Add a `UM Administrator` role, with specific permissions.
+- Add an email address field accesible to user management role.
+- Allow to send an e-mail for new registrations to the email adress added above
+, if any.
+- Add a `hook_um_administrator_grant_permissions` to grant new priviledges to
+the UM adminsitrator role.
+
+
+## V 1.1.0
+https://citnet.tech.ec.europa.eu/CITnet/jira/browse/NEPT-2853
+- Users who can access the user management view (admin and user management
+role) can view roles that are already assigned to users.
+- Add a view to allow seeing for the selected users, the added (in green),
+removed roles (in red) and resulting state (in blue).
+- Do not allow to add and remove the same role at once.
+
+## V 1.0.0
+https://citnet.tech.ec.europa.eu/CITnet/jira/browse/NEPT-2830
+
+- Create 'User management' role, to be granted to a 'user manager'. This person
+can add or remove roles to/from users.
+- Create a 'user management' page where users with 'user management role' can
+administer roles.
+- Create variables 'nexteuropa_user_management_banned_roles' and
+'nexteuropa_user_management_banned_role_ids' to list roles that cannot be
+granted to the users by the user manager (blacklist).
+
+# Permissions restrictions
+To ensure more secure websistes, a series of permissions have been restricted
+from usage in the platform.
+## Forbidden permissions
+The list of permissions forbidden are listed in the "Secure Drupal development
+at the European Commission" report available on the OPENEU wiki :
+
+-  'administer content types'
+-  'administer ckeditor_lite'
+-  'acccess all views'
+-  'administer ecas'
+-  'administer features'
+-  'administer fields'
+-  'administer file types'
+-  'administer filters'
+-  'administer jquery update'
+-  'administer modules'
+-  'administer om maximenu'
+-  'administer page manager'
+-  'administer permissions'
+-  'administer site configuration'
+-  'administer software updates'
+-  'administer themes'
+-  'administer users'
+-  'administer views'
+-  'bypass file access'
+-  'bypass node access'
+-  'bypass rules access'
```

Comment:
I think here you need to list then all the 30 forbidden permissions, not just the ones that come from the platform, due all are forbidden both by the toolkit and the mentioned document, and actually the module needs to check it, therefore.

Reply:
No we don't, only the platform's permissions. Toolkit is blocking those already.
This module will be moved to platform once final, and platform cannot update its code each time there is a forbidden permission to add, for non platform code.  Toolkit should.

Reply:
The module is also meant to be used by non platform sites, so please keep all permissions which are marked forbidden instead of making the module platform-exclusive.